### ISSUE TITLE
Potential BA.2.3.17/BA.2.75.2 recombinant (18 seqs as of 2022-10-17)

### ISSUE THREAD

**Issue by c19850727** [cov-lineages/pango-designation#1229]
I am proposing this potential recombinant with only 18 sequences based on its geographical distribution and potential growth advantage during recent few days.

### Description
**Recombinant between**: BA.2.3.17 and BA.2.75.2
**Earliest sequence**: 2022/07/27 (Nepal)
**Most recent sequence**: 2022/10/10 (Singapore ex-Nepal)
**Countries circulating**: Nepal (2/17), Brunei (3/17), UK-England (1/17), France (5/17),  Germany (1/17), Israel (2/17), Malaysia (1/17), Singapore ex-Nepal (1/17), Singapore local (1/17) 

**Potential breakpoint**: somewhere between NSP12 and Spike NTD (15451-22000)
**Potential BA.2.3.17 donor**: EPI_ISL_12874157 or EPI_ISL_12788684 or EPI_ISL_12788514 from US-CA or US-HI.

**Mutation profile**:
![image](https://user-images.githubusercontent.com/86835836/196317999-d0028c6b-a0c2-48ff-bdeb-63c63a3ce8c8.png)
**Private mutations**: none
**GISAID query**: Spike_D1199N, NSP12_A97V, -NSP8_N118S, -NSP3_P822S, -NSP16_G671S (there are 2 unrelated random sequences picked up by this query, both from early 2021, so it's easily identifiable by dates of collection)
**Cov-spectrum query**: T3796C, T3927C, T4586C, T5183C, G12444A, A15451G, C13730T, C7528T, G25157A

### Evidence
**Usher tree:**
![image](https://user-images.githubusercontent.com/86835836/196320139-aaf3da15-5070-43fa-b0cb-34260a50d47c.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_db2c_dffb30.json?branchLabel=aa%20mutations&c=gt-nuc_3796&label=nuc%20mutations:T3796C,T3927C,T4586C,T5183C,G12444A,A15451G

### Genomes:
EPI_ISL_14807682, EPI_ISL_14959369, EPI_ISL_14960037, EPI_ISL_15031974, EPI_ISL_15136400, EPI_ISL_15140939, EPI_ISL_15244536-15244537, EPI_ISL_15244551, EPI_ISL_15296261, EPI_ISL_15313055, EPI_ISL_15325673, EPI_ISL_15330221, EPI_ISL_15335186, EPI_ISL_15384119, EPI_ISL_15385024, EPI_ISL_15385056-15385057

**Comment by c19850727** [comment]
19 sequences as of 2022-10-19. Newly found in Malaysia.

**Comment by c19850727** [comment]
20 sequences as of 2022-10-20.

**Comment by c19850727** [comment]
31 sequences as of 2022-10-22. 10 from Brunei and 1 from Nepal.

**Comment by FedeGueli** [comment]
E_T11A,Spike_D1199N, NSP12_A97V, -NSP8_N118S, -NSP3_P822S, -NSP16_G671S finds 28 all belonging this lineage.

**Comment by c19850727** [comment]
31 sequences as of 2022-10-24. Newly found in Australia.

BTW @FedeGueli some of these genomes have dropouts in E protein so I didn't use E_T11A in my query.

**Comment by corneliusroemer** [comment]
Thanks @c19850727, I agree this one looks interesting due to geographic spread and growth

**Comment by c19850727** [comment]
34 genomes. 3 more from Brunei.

**Comment by c19850727** [comment]
yeah Fede 2 more from Germany emerged after my daily coffee-time GISAID screening -o-

**Comment by c19850727** [comment]
41 sequences as of 2022-10-30. Newly found in France, and 3 more from Brunei.

**Comment by FedeGueli** [comment]
i see one more from Malaysia and one more from Japan

**Comment by Sam8138** [comment]
The entry of BE.1.4.4 incorrectly lists the alias as: B.1.1.529.5.3.1.1.4.3

It should be: B.1.1.529.5.3.1.1.4.4

**Comment by AngieHinrichs** [comment]
Thanks for reporting the typo @Sam8138, fixed now.

**Comment by Sam8138** [comment]
In the entry for BA.2.23.1:

Alias of Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532

Should read: ("Alias of " is duplicated)

Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532

**Comment by FedeGueli** [comment]
> In the entry for BA.2.23.1: Alias of Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532 Should read: ("Alias of " is duplicated) Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532

is this related to this issue?

**Comment by FedeGueli** [comment]
i found 45 sequences as today with Spike_T478K,Spike_D1199N, NSP12_A97V, -NSP8_N118S, -NSP3_P822S, -NSP16_G671S

**Comment by Sam8138** [comment]
In the entry for BA.2.23.1:

Alias of Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532

Should read: ("Alias of " is duplicated)

Alias of B.1.1.529.2.23.1, mainly found in England, USA and Denmark, from pango-designation issue #532




In entry for CU.1:

Alias of BA.5.1.529.5.1.26.1, Central/Eastern Europe, S:T95I, S:T547K

Should read:

Alias of B.1.1.529.5.1.26.1, Central/Eastern Europe, S:T95I, S:T547K

**Comment by AngieHinrichs** [comment]
@Sam8138 thanks for reporting the typos.  If you find any more, please open a new issue: 

https://github.com/cov-lineages/pango-designation/issues/new

That will help to keep the discussion in this issue focused on the potential recombinant lineage.

**Comment by c19850727** [comment]
46 sequences as of 2022-11-03. Newly found in Japan (local case).

**Comment by FedeGueli** [comment]
47seqs as today

**Comment by FedeGueli** [comment]
51 as today

**Comment by c19850727** [comment]
53 as of 2022-11-07. Newly found in Turkey.

**Comment by c19850727** [comment]
63 as of 2022-11-08. Another big upload from Brunei (10/75).

**Comment by c19850727** [comment]
64 sequences as of 2022-11-10.

**Comment by InfrPopGen** [comment]
Thanks for submitting. We've added recombinant lineage XBH with 36 newly designated sequences, and 0 updated designations. Defining mutations C7528T (ORF1b:A88V), nt:C13730T (following BA.2.75.2 G25157A (S:D1199N) on current tree).
